### ISSUE TITLE
Error on Cloudflare Worker

### ISSUE THREAD

**Issue by kleinpetr** [Remove-Background-ai/rembg.js#9]
Hi,

I am trying to use `rembg` with my Nuxt application on CF Worker, 

`[request error] [unhandled] [500],e.addListener is not a function`

So I've enabled `nodejs_compat_v2` as it's described here https://developers.cloudflare.com/workers/runtime-apis/nodejs/eventemitter/

But still getting the same error.
Thanks for help

**Comment by mtrabelsi** [comment]
Hi thanks for the report.

can you show some written code?

At a first look, it looks like Cloudflare integration issue/worker issue, the library supports mjs files and should run fine in environments compatible with modern JavaScript modules.

However, to know the root cause you can skip the library and you can copy code functionality from here https://github.com/Remove-Background-ai/rembg.js/blob/main/index.ts directly into your code without Node.js import, and give it a try.

please report the result back to me!

**Comment by kleinpetr** [comment]
Hi @mtrabelsi thanks for the reply, this is exactly what I did, since I need base64 only, I could simplify the entire logic. So I guess the problem is within using `fs` in the package.

**Comment by mtrabelsi** [comment]
Yes indeed the fs is the issue, I might need to remove tmp-promise and related features (ex: like saving to disk) as this Base64 is already enough for most usecases, I will keep this issue open until this is fixed
